changelog.asc

```diff
@@ -2,7 +2,8 @@
 :sectanchors:
 = AnkiDroid Changelog
 
-== Version 2.8.5 (2018-10-12)
+== Version 2.8.5beta1 (2018-10-12)
```

Comment:
I'd rather just remove 2.8.5 from the release notes altogether. You can post an explanation on the forum if you feel it's necessary.

Reply:
Yeah - agreed there